packages/v4/patternfly-docs/content/design-guidelines/content/writing-design-documentation.md

```diff
@@ -101,34 +128,3 @@ If you're linking to other PatternFly pages, use relative URLs instead of full U
 Relative URL: /components/tooltip/design-guidelines
 Full URL: https://www.patternfly.org/v4/components/tooltip/design-guidelines
```

Comment:
Need to add an extra empty line between these two lines cause they're showing up on the same line right now. Also would bold "Relative URL:" and "Full URL:"

<img width="534" alt="image" src="https://user-images.githubusercontent.com/51165119/191105231-3b704c43-d1df-4423-b2b1-82128b5dafdc.png">